pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1360,6 +1360,10 @@ self: super: {
     extraPrefix = "";
   });
 
+  logging-effect = markUnbroken (super.logging-effect.override {
+    unliftio-core = super.unliftio-core_0_2_0_1;
+  });
```

Comment:
Is this for ZHF?  If so, then it is basically fine as it is (assuming you will just commit straight to the `release-20.09` branch).

If this isn't for ZHF, and you just want this fixed in `master`, can you do the following:

1. Instead of using `markUnbroken` in this file, remove `logging-effect` from the `broken-packages` list in `configuration-hackage2nix.nix`.
2. Change the base branch on github to `haskell-updates` (and probably rebase on `haskell-updates` as well).

The `hackage-packages.nix` file gets regenerated daily, and since you've removed `logging-effect` from the list of broken packages, next time it is regenerated, it will be regenerated without `broken = true;`.

Reply:
Not directly but it will help with ZHF and can be backported to 20.09.

Now that I have played with the patch more, I think a better solution is to pin the version to `logging-effect ==1.3.10` because otherwise, it will not work with those other libraries.

Reply:
Will there be more updates to this pull request?